lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+## Supported Gerrit Versions
+
+All Gerrit 3.x Versions should be working.
+
+The current implementation uses the "hashtags" Feature from Gerrit and therefore needs a Gerrit version with [NoteDB](https://gerrit-review.googlesource.com/Documentation/note-db.html) backend.
```

Comment:
```suggestion
The current implementation uses Gerrit's "hashtags" feature.
Therefore you must use a Gerrit version that uses the [NoteDB](https://gerrit-review.googlesource.com/Documentation/note-db.html) backend.
```

How about something like this? Please check if this is correct before applying!

Do we mean:

- You (as Gerrit user) must use NoteDB with Gerrit, or
- You (as Gerrit user) must use a version of Gerrit that includes NoteDB, or
- Something else?

Reply:
> You (as Gerrit user) must use a version of Gerrit that includes NoteDB

That's the requirement.